addons/website/static/src/widgets/website_editor/website_editor.js

```diff
@@ -37,14 +37,18 @@ export const WebsiteEditorWidget = AbstractAction.extend({
      * @override
      */
     start() {
-        this.iframe = this.el.querySelector('iframe');
+        const iframeEls = this.el.querySelectorAll('iframe');
+        this.iframe = iframeEls[1];
+        this.iframe2 = iframeEls[0];
         const currentWebsiteId = this.websites.find(website => website.selected).id;
         this.iframe.src = `website/force/${currentWebsiteId}?path=${this.params.path}`;
+        this.iframe2.src = this.iframe.src;
 
         this.iframe.addEventListener('load', () => {
             const iframeUrl = this.iframe.contentDocument.location.href;
             if (document.location.href !== iframeUrl) {
                 history.pushState({}, 'Website Editor', iframeUrl);
+                this.iframe2.src = this.iframe.src;
```

Comment:
I tested and it works well :) I just pushed a rebased version of the commit with this change: `this.iframe2.src = iframeUrl;`
Having a fallback with the same content as the iframe seems to be the solution, I will continue to test and try to find a way to reduce the http calls

Reply:
:+1: For the redirection flicker on Chrome (frontend -> backend), don't waste days on it if this is a problem. At worst we'll move to a python redirection (and maybe this is simply the way to go instead JS anyway) , but it's better to wait for the current httppocalypse which is being developed. And at best AL could convince FP that we simply do not want that.

Reply:
So focus on the iframe navigation which is critical to me :wink: We need the iframe for multiple features... so if we cannot make that work, that would be a huge problem :grimacing: